similarity_api/runtime.txt

```diff
@@ -0,0 +1 @@
+python-3.9.18
```

Comment:
This does not match other versions of python used.

https://github.com/cloudfoundry/python-buildpack/releases
It looks like the latest python build pack no longer supports 3.9.17. So 3.9.18 or 3.10.13 or 3.11.5 should be used.

I think 3.10.13 looks to be the newest stable version. as 3.11.5 is still in bug fix according to https://devguide.python.org/versions/

Reply:
could this value be detected from within the env or some other method so it only needs to be set/defined in 1 location. That way we never get a mixed version issue popping up.

Reply:
Answered this above ^.